documentation/service_admin/proc-creating-project.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-installing-manual-steps.adoc
+
+include::common/attributes.adoc[]
+
+:cmdcli: oc
+
+[id='creating-project_{context}']
+= Creating the project for {ProductName}
+
+ifeval::["{cmdcli}" == "oc"]
+. Create the `enmasse` project:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} new-project enmasse
+----
+endif::[]
+ifeval::["{cmdcli}" == "kubectl"]
+
+. Create the `enmasse` namespace:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create namespace enmasse
+----
+
+. Set the `enmasse` namespace as the default namespace:
++
+[options="nowrap",subs="attributes"]
+----
+kubectl config set-context $(kubectl config current-context) --namespace=enmasse
```

Comment:
Why isn't this one using the cmdcli attribute?  Also, there might be a better way to handle this, let me think on it.